### ISSUE TITLE
Camera database

### ISSUE THREAD

**Issue by chendeni** [SuperHackio/LaunchCamPlus#17]
I have an idea for a new feature, the SMG Camera Database.  
Since every camera type works in different and unique ways, a database would make camera editing much easier especially for those new to SMG2 modding.  Similar to the SMG Object Database, the camera database documents every camera type and their properties, and explains how they work.  For example, some cameras use the absolute coordinates of the zone while some use offsets from the object that uses the camera.  Some cameras ignore certain attributes, and some are only compatible with certain objects.

**Comment by SuperHackio** [comment]
So.... basically what the Unique Editors are for?

Or do you refer to something else?

**Comment by chendeni** [comment]
The SMG Camera Database is not part of Launch Cam Plus itself.  It is a database that would work just like the Object Database to be used as a reference or guide on how each camera type works.  It will be useful for camera editing, development of new Launch Cam Plus features such as more unique editors, and other modding that involves cameras.

**Comment by SuperHackio** [comment]
So... a list that contains every camera from both Galaxy games?

**Comment by chendeni** [comment]
Yes, a list that contains every camera type from the Galaxy games.  Includes a description, how its fixpoint works, which variables are used or ignored, and much more.
